Dockerfile.ubi

```diff
@@ -168,7 +189,7 @@ FROM vllm-openai as vllm-grpc-adapter
 USER root
 
 RUN --mount=type=cache,target=/root/.cache/pip \
-    pip install vllm-tgis-adapter
+    pip install vllm-tgis-adapter==0.1.3
```

Comment:
With respect to raised blocker issue RHOAIENG-9356, only we need to upgrade the reference of `vllm-tgis-adapter==0.1.3`. What other changes are for?

Reply:
Required by IBM for model encryption.

Reply:
https://github.com/opendatahub-io/vllm/pull/64